### PR TITLE
Create Franchisee Table

### PR DESCRIPTION
**Pull request by vrindavnair**

### CODE REVIEW

File: src/components/FranchiseComponents/FranchiceeReuseableComponents/FranchiseeCard.jsx
Comment (JaseelaAlimahamed) [inline src/components/FranchiseComponents/FranchiceeReuseableComponents/FranchiseeCard.jsx:1]: vrinda you are creationg admin page components then why did you save in franchise components.......
make the folder path like component/adminComponents/FanchiseManagement/franchiseeCard

File: src/components/FranchiseComponents/FranchiceeReuseableComponents/FranchiseeIcons/Email.png
Comment (JaseelaAlimahamed) [inline src/components/FranchiseComponents/FranchiceeReuseableComponents/FranchiseeIcons/Email.png:1]: these are not components set these icons in assets or public file
assets/adminAssets/your file

File: src/components/FranchiseComponents/FranchiceeReuseableComponents/FranchiseeTable.jsx
Comment (JaseelaAlimahamed) [inline src/components/FranchiseComponents/FranchiceeReuseableComponents/FranchiseeTable.jsx:1]: change path

File: src/components/FranchiseComponents/FranchiseDetails/FranchiseeList.jsx
Comment (JaseelaAlimahamed) [inline src/components/FranchiseComponents/FranchiseDetails/FranchiseeList.jsx:1]: change path

**Review by JaseelaAlimahamed — CHANGES_REQUESTED** [review]

File: src/components/FranchiseComponents/FranchiceeReuseableComponents/FranchiseeCard.jsx
Comment (vrindavnair) [reply src/components/FranchiseComponents/FranchiceeReuseableComponents/FranchiseeCard.jsx:1]: Implemented as requeated

File: src/components/FranchiseComponents/FranchiceeReuseableComponents/FranchiseeIcons/Email.png
Comment (vrindavnair) [reply src/components/FranchiseComponents/FranchiceeReuseableComponents/FranchiseeIcons/Email.png:1]: Implemented as requested

File: src/components/FranchiseComponents/FranchiceeReuseableComponents/FranchiseeTable.jsx
Comment (vrindavnair) [reply src/components/FranchiseComponents/FranchiceeReuseableComponents/FranchiseeTable.jsx:1]: Implemented as requested

File: src/components/FranchiseComponents/FranchiseDetails/FranchiseeList.jsx
Comment (vrindavnair) [reply src/components/FranchiseComponents/FranchiseDetails/FranchiseeList.jsx:1]: Implemented as requested

**Pull request merged**